.devcontainer/java/devcontainer.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "Java",
+  "image": "mcr.microsoft.com/devcontainers/javascript-node:20",
```

Comment:
Use prebuilt version of java

Reply:
Change image to "mcr.microsoft.com/devcontainers/java:17"